numpy/random/_bounded_integers.pyx

```diff
@@ -1183,7 +1183,7 @@ cdef object _rand_bool(object low, object high, object size,
                 random_bounded_bool_fill(state, off, rng, 1, use_masked, &out_val)
             return np.bool_(<bool_t>out_val)
         else:
-            out_arr = <np.ndarray>np.empty(size, np.bool)
+            out_arr = <np.ndarray>np.empty(size, np.bool_)
```

Comment:
This line is somehow running as `np.bool` in CI, even though it clearly says `np.bool_` here...

Reply:
Turns out that #14898 was confusing me here